src/components/app/ProfileLoader.js

```diff
@@ -53,9 +54,13 @@ class ProfileLoaderImpl extends PureComponent<Props> {
       retrieveProfilesToCompare,
     } = this.props;
     switch (dataSource) {
-      case 'from-browser':
-        retrieveProfileFromBrowser();
+      case 'from-browser': {
```

Comment:
I'd be surprised this path works at all anyway :-)

Currently I see we forbid moving from /public to /from-browser. And even if we wouldn't, we'd still not have anything to capture from the browser.

But in the future if we use `ProfileLoader` for the startup path too this will be used of course.

Reply:
Yeah the path probably doesn't work, I didn't test it. But, as you said, we may switch to it in the future so I thought it would be reasonable to include this change.